lib/features/family/features/home_screen/presentation/pages/parent_home_screen.dart

```diff
@@ -64,17 +65,24 @@ class _ParentHomeScreenState extends State<ParentHomeScreen> {
     }
   }
 
-  void _onProfileSwitchPressed(BuildContext context) {
-    context.pop();
-    AnalyticsHelper.logEvent(
-      eventName: AmplitudeEvents.profileSwitchPressed,
-    );
-  }
-
   Widget _giveTile(BuildContext context) => Padding(
         padding: const EdgeInsets.fromLTRB(24, 24, 24, 0),
         child: FunTile(
-          onTap: () => context.pushNamed(FamilyPages.giveByListFamily.name),
+          onTap: () {
```

Comment:
In the game this is done at the moment you really are giving (on/after the org information bottomsheet I think). Wouldn't that be a better place? :)

Reply:
I'm also fine with this. So i'll accept the PR. Up to you :)